.github/workflows/s2i.yaml

```diff
@@ -9,6 +9,8 @@ on:
     paths:
       - .github/workflows/s2i.yaml
       - s2i/**
+  schedule:
+    - cron: "0 0 1 * *"
```

Comment:
If I understand correctly, the s2i action is being deprecated. Hence, should the workflow for it be deleted?